packages/@azure/servicebus/data-plane/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@azure/amqp-common": "^1.0.0-preview.1",
+    "@azure/ms-rest-js": "^1.8.0",
```

Comment:
Sorry, I know we have gone through this once before.... but what is this dependency for?
In this PR, we have not yet updated the amqp common library, so I don't see why we need to change/add any dependencies related to ms-rest